parquet-hadoop/src/main/java/org/apache/parquet/hadoop/rewrite/RewriteOptions.java

```diff
@@ -64,15 +72,53 @@ private RewriteOptions(Configuration conf,
     this.indexCacheStrategy = indexCacheStrategy;
   }
 
+  /**
+   * Gets the {@link Configuration} part of the rewrite options.
+   *
+   * @return the associated {@link Configuration}
+   */
   public Configuration getConf() {
+    return ConfigurationUtil.createHadoopConfiguration(conf);
+  }
+
+  /**
+   * Gets the {@link ParquetConfiguration} part of the rewrite options.
+   *
+   * @return the associated {@link ParquetConfiguration}
+   */
+  public ParquetConfiguration getParquetConfiguration() {
     return conf;
   }
 
-  public List<Path> getInputFiles() {
```

Comment:
I think we should not remove a public method directly.

Reply:
It seems japicmp didn't pick it up as the erasure is the same as in `List<InputFile> getInputFiles()`. I'll restore the method.